pallets/committee/src/lib.rs

```diff
@@ -141,12 +147,15 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
 		fn build(&self) {
+			let voting_period = T::VotingPeriod::get();
 			for member in &self.council_members {
 				Members::<T>::insert(member, MemberType::Council);
+				VotingEligibility::<T>::insert(member, voting_period);
```

Comment:
This should be zero so they're instantly able to vote.
And should be documented: Genesis members can vote instantly.
```suggestion
				VotingEligibility::<T>::insert(member, 0);
```